### PR TITLE
Merge commits to support background NSURLSession uploads and downloads

### PR DESCRIPTION
**Pull request by boxtcarpel**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @boxtcarpel has signed the CLA. Thanks for the pull request!

File: BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleAppDelegate.m [hunk 1]
```diff
@@ -138,8 +63,12 @@ - (void)application:(UIApplication *)application handleEventsForBackgroundURLSes
 {
     NSLog(@"handleEventsForBackgroundURLSession identifier %@", identifier);
     //FIXME: Need to get the BOXContentClient for the currently logged in user and set up its URL session manager
```
Comment (boxtcarpel) [inline BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleAppDelegate.m]: We can kill this now, right? because it's a global singleton?

**Comment by irvingruan** [comment]
+1

**Comment by thuyBox** [comment]
+2 with following up PR https://github.com/box/box-ios-sdk/pull/356

File: BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleAppDelegate.m (hunk 1 above)
Comment (thuyBox) [reply BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleAppDelegate.m]: yes, removed in following up PR

**Comment by thuyBox** [comment]
+2

**Pull request merged**